usecases/schema/helpers_for_test.go

```diff
@@ -14,18 +14,15 @@ package schema
 import (
```

Comment:
I would suggest renaming the file from helpers_for_test.go to helpers_test.go since it is implied by the context that mocks are intended for testing